test/analysis/SmartDetectorsAnalysisTests/AlertPresentationTests.cs

```diff
@@ -54,7 +54,15 @@ public void WhenProcessingAlertThenTheContractsAlertIsCreatedCorrectly()
             VerifyPresentationTestAlertDisplayedProperty(contractsAlert.AlertProperties, "KeyValueWithHeaders", "KeyValueWithHeadersDisplayName", 4);
             VerifyPresentationTestAlertDisplayedProperty(contractsAlert.AlertProperties, "Table", "TableDisplayName", 5);
             VerifyPresentationTestAlertDisplayedProperty(contractsAlert.AlertProperties, "SingleColumnTable", "SingleColumnTableDisplayName", 6);
-            VerifyPresentationTestAlertDisplayedProperty(contractsAlert.AlertProperties, "DataPoints", "ChartDisplayName", byte.MaxValue);
+            VerifyPresentationTestAlertDisplayedProperty(contractsAlert.AlertProperties, "Name1", "First name title", 7);
```

Comment:
>Name1 [](start = 90, length = 5)

that's not what i meant, and it will cause problems in the rule engine
we need some sort of hierarchical notation here: AdditionalData.Item1.Name1